blob/service.go

```diff
@@ -86,6 +89,47 @@ func DefaultSubmitOptions() *SubmitOptions {
 	}
 }
 
+type BlobsubResponse struct {
+	blobs  []*Blob
+	height uint64
+}
+
+func (s *Service) Subscribe(ctx context.Context, ns share.Namespace) (<-chan *BlobsubResponse, error) {
+	headerCh, err := s.headerSub(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	blobCh := make(chan *BlobsubResponse)
```

Comment:
A little buffer wouldn't hurt and will lead to less context switching.

Reply:
done